qmlti/src/resources/delivery_odata/DeliveryOdataService.php

```diff
@@ -0,0 +1,69 @@
+<?php
+/**
+ * Copyright (C) 2016 Questionmark Computing Limited.
+ * License GNU GPL version 2 or later (see LICENSE.TXT file)
+ * There is NO WARRANTY, to the extent permitted by law.
+ */
+
+require_once "LTIRestClient.php";
+
+/**
+ * Class DeliveryOdataService
+ *
+ * @package Questionmark\qm_services\Api
+ */
+class DeliveryOdataService  {
+
+  private $ServiceEndpoint = '';
+  private $ServiceName = 'Delivery Odata Service';
+  private $RestClient = NULL;
+
+  public function __construct($customer_id, $url, $qmwise_username, $qmwise_password) {
+    $this->RestClient = new LTIRestClient($customer_id, $url, $qmwise_username, $qmwise_password);
+    $this->ServiceEndpoint = $url;
+  }
+
+  // Assessments FEED
+  function GetAssessment($id = null) {
```

Comment:
Missing PHPDoc. Probably a good idea if someone else has to maintain this in the future. Repeat for other functions below this one for this file.